aten/src/ATen/core/function_schema.h

```diff
@@ -518,7 +518,7 @@ inline std::ostream& operator<<(std::ostream& out, const Argument& arg) {
       auto default_val = arg.default_value().value().toIntList();
       if (default_val.size() > 1) {
         auto all_defaults_the_same = true;
-        for (const auto& i : c10::irange(1, default_val.size())) {
+        for (const auto i : c10::irange(1, default_val.size())) {
```

Comment:
Maybe even change `auto` to `int` or whatever  type it is for extra clarity?

Reply:
I want compiler to decide whether it should be `int`, `unsigned int`, `long` or `unsigned long`, but results of `irange` should never be captures